.eslintrc

```diff
@@ -21,5 +21,6 @@
     "no-use-before-define": ["warn"],
     "react/no-multi-comp": ["warn"],
     "react/jsx-no-bind": ["warn"],
+    "react/forbid-prop-types": ["off"]
```

Comment:
yeah good point. I disabled this because it was restricting me from using array prop types in the client list and client panel components. I suppose I could leave this rule on and find a different solution?